verification/spec/connection-handshake/Environment.tla

```diff
@@ -0,0 +1,233 @@
+---------------------------- MODULE Environment ----------------------------
+
+EXTENDS Naturals, FiniteSets, Sequences
+
+
+CONSTANT MaxHeight,     \* Maximum height of any chain in the system.
+         MaxBufLen      \* Length (size) of message buffers.
```

Comment:
Can you add a `verification/spec/connection-handshake/README.md` and describe a bit the content and also recommended values for the constants above? I put in some numbers and my TLA+ toolbox is pretty much dead...will try to restart my laptop now. I wish some good engineers would redesign this and make it a modern and reliable tool :(

Reply:
I am not able to run it locally either, even with values `2` and `2` had issues with model checking.
I'm trying to refurbish the code to eliminate some of the non-determinism.

Reply:
yes, I tried the same and something is wrong. Never saw this before, the toolbox to just hang and crash. not sure at what point this started, it was fast last time I tried but it was days ago and there was only the `openInit` there.

Reply:
Quick fix when the toolbox hangs repeatedly for me was to delete the model and create another one!

Reply:
We should probably switch to apalache from the command line.

Reply:
> Quick fix when the toolbox hangs repeatedly for me was to delete the model and create another one!

didn't work for me, tried many times

Reply:
> We should probably switch to apalache from the command line.

maybe. but i think it's a problem with the model. For this spec and small constants I expect local TLA toolbox to be fine. I can run fastsync with long chains and it's fine even if it takes ~10mins.

Reply:
I think it's because I'm using the `ENABLED` operator. I'm trying to refactor that part now...

Reply:
Pushed a new variant today that no longer has the hanging issue, after I removed the `ENABLED` operator.